### PR TITLE
Add doctest for Series.transform_batch with np.ufunc parameter

### PR DESCRIPTION
**Pull request by beobest2**
image

Adding a missing test for using the ``np.ufunc`` parameter of ``Series.transfor_batch``.

### CODE REVIEW

**Comment by beobest2** [comment]
Unexpectedly, the coverage of the targeted line is not filled. I guess I should add unittest.

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -2883,12 +2883,12 @@ def transform_batch(self, func, *args, **kwargs) -> "ks.Series":
 
         Examples
         --------
-        >>> df = ks.DataFrame([(1, 2), (3, 4), (5, 6)], columns=['A', 'B'])
+        >>> df = ks.DataFrame([(1, -2), (3, -4), (5, -6)], columns=['A', 'B'])
```
Comment (itholic) [inline databricks/koalas/series.py:2886]: Let's don't try to fix the existing tests as possible.

I think maybe you can create another `Series` for test `np.abs`, or simply just test with other `np.ufunc` just like the below if there is no special reason to should test the `np.abs`.

```python
>>> df.A.transform_batch(np.add, 10)
0    11
1    13
2    15
Name: A, dtype: int64
```

**Comment by itholic** [comment]
> Unexpectedly, the coverage of the targeted line is not filled. I guess I should add unittest.

I think it is not that matter if the coverage increased or not.
I think this test more fit for doctest rather unittest because this test let users know about how to use the `transform_batch` with `np.ufunc`.

**Comment by beobest2** [comment]
> > Unexpectedly, the coverage of the targeted line is not filled. I guess I should add unittest.
> 
> I think it is not that matter if the coverage increased or not.
> I think this test is more fit for doctest rather unittest because the purpose of this test is letting users know how to use the `transform_batch` with `np.ufunc`.


I agree with it. I modified it as you said.

**Comment by itholic** [comment]
Looks okay to me

**Comment by ueshin** [comment]
The build must be fixed.
@beobest2 Could you submit and push an empty commit to re-run the tests?

**Pull request merged**